markets/perps-market/contracts/storage/PerpsMarket.sol

```diff
@@ -280,4 +293,15 @@ library PerpsMarket {
             }
         }
     }
+
+    /**
+     * @dev Returns the market debt incurred by all positions
+     * @notice  Market debt is the sum of all position sizes multiplied by the price, and old positions pnl that is included in the debt correction accumulator.
+     */
+    function marketDebt(Data storage self, uint price) internal view returns (int) {
+        // all positions sizes multiplied by the price is equivalent to skew times price
+        // and the debt correction accumulator is the sum of all positions pnl
+
+        return self.skew.mulDecimal(price.toInt()) - self.debtCorrectionAccumulator;
+    }
```

Comment:
```suggestion
    function marketDebt(Data storage self, uint price) internal view returns (int) {
        // all positions sizes multiplied by the price is equivalent to skew times price
        // and the debt correction accumulator is the sum of all positions pnl
        int traderUnrealizedPnl = self.skew.mulDecimal(price.toInt()) - self.debtCorrectionAccumulator;
        int totalAccruedFunding = self.skew.mulDecimal(price.toInt()).mulDecimal(currentFundingRate();

        return traderUnrealizedPnl + totalAccruedFunding;
    }
```

Reply:
this will take care of unrealized funding.  if we subtract it out of the accumulator then it would be counted double, so we only use trader pnl in the accumulator.